config/confighttp/README.md

```diff
@@ -20,6 +20,9 @@ README](../configtls/README.md).
 - [`read_buffer_size`](https://golang.org/pkg/net/http/#Transport)
 - [`timeout`](https://golang.org/pkg/net/http/#Client)
 - [`write_buffer_size`](https://golang.org/pkg/net/http/#Transport)
+- `compression`: Compression type to use among `zstd`, `snappy`, `gzip`, `zlib`, and `deflate`.
```

Comment:
I think we decided on a shorter list to support in gRPC why do we extend here?

Reply:
https://github.com/open-telemetry/opentelemetry-collector/blob/main/config/configgrpc/configgrpc.go#L50

Reply:
`zlib` and `deflate` were already supported in otlphttpexporter. I kept it after discussed with @Aneurysm9.